api-spec/WFS3core+STAC.yaml

```diff
@@ -1,842 +1,909 @@
-openapi: 3.0.1
-info:
-  title: STAC API integrated with OGC Web Feature Service standard
-  version: 0.5.1
-  description: >-
-    This is an example OpenAPI specification of a SpatioTemporal Asset Catalog

```

Comment:
I think we should put the title and description overrides back into the WFS fragment.

Reply:
Do you still want to do that or is that a todo for later?